demisto_sdk/commands/create_artifacts/content_artifacts_creator.py

```diff
@@ -283,6 +286,12 @@ def add_layout_as_content_item(self, content_object: ContentObject):
                 'name': content_object.get('name', '')
             })
 
+    def add_pre_process_rules_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.PRE_PROCESS_RULES].append({
+            'name': content_object.get('name') or content_object.get('id', ''),
+            'description': content_object.get('description', '')
```

Comment:
according to your schema, the file does not have a `description` field

Reply:
https://github.com/demisto/demisto-sdk/commit/ef62252ee345d3dc9cdd5edce4098abb45d4abac

Reply:
After a discussion with @Bargenish, we're keeping it.